### ISSUE TITLE
Reverse confusing current logic of SKIP_PYCURL_TESTS

### ISSUE THREAD

**Issue by koobs** [stripe/stripe-python#172]
The current logic of SKIP_PYCURL_TESTS (environment variable) is the reverse of what it should be (to make sense):

stripe/test/test_integration.py:
```
        if not os.environ.get('SKIP_PYCURL_TESTS'):
            self.skipTest('Pycurl skipped as SKIP_PYCURL_TESTS is set')
```

At the moment, setting SKIP_PYCURL_TESTS to any value (or just defining it) enables the tests.

Options:

 * Rename variable -> DONT_SKIP_PYCURL_TESTS
 * Replace ```if not``` conditional with a plain ```if``` and don't set the variable in tox.ini (to run the tests).

**Comment by kyleconroy** [comment]
Fix #172

**Comment by kyleconroy** [comment]
Fixed: https://github.com/stripe/stripe-python/pull/178
